dil/src/main/java/org/assimbly/dil/transpiler/transform/Transform.java

```diff
@@ -47,7 +47,7 @@ private static String camel2ToCamel3(String input){
         map.put("xml2excel","xmltoexcel");
         map.put("excel2xml","exceltoxml");
         map.put("csv2xml","csvtoxml");
-        map.put("global-variables","globalvariables");
+        map.put("global-variables","tenantvariables");
```

Comment:
Maybe good to add:

map.put("tenant-variables","tenantvariables");

Also

Reply:
Ok, I can add that one also.
But for now, we gonna still use global-variables on the frontend, until we use the old backend.